packages/cubejs-databricks-jdbc-driver/src/helpers.ts

```diff
@@ -15,26 +15,37 @@ async function fileExistsOr(
 
 export async function resolveJDBCDriver(): Promise<string> {
   return fileExistsOr(
-    path.join(process.cwd(), 'DatabricksJDBC42.jar'),
+    path.join(process.cwd(), `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
     async () => fileExistsOr(
-      path.join(__dirname, '..', 'download', 'DatabricksJDBC42.jar'),
+      path.join(__dirname, '..', 'download', `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
       async () => {
         const pathOrNull = await downloadJDBCDriver();
         if (pathOrNull) {
           return pathOrNull;
         }
         throw new Error(
-          'Please download and place DatabricksJDBC42.jar inside your ' +
+          `Please download and place databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar inside your ` +
           'project directory'
         );
       }
     )
   );
 }
 
-export function extractUidFromJdbcUrl(jdbcUrl: string): string {
-  const { pathname } = new URL(jdbcUrl);
-  const [_, ...params] = pathname.split(';');
-  const searchParams = new URLSearchParams(params.join('&'));
-  return searchParams.get('UID') || 'token';
+/**
+ * Extract if exist UID and PWD from URL and return UID, PWD and URL without these params
+ * @param jdbcUrl
+ */
+export function extractAndRemoveUidPwdFromJdbcUrl(jdbcUrl: string): [string, string, string] {
+  const uidMatch = jdbcUrl.match(/UID=([^;]*)/);
```

Comment:
Can you add link/reference/something to this drivers specific parsing?
I can't give you a link to GitHub, because my IP is banned in https://github.com/databricks/databricks-jdbc  repo
But there's `buildPropertiesMap` in com/databricks/jdbc/api/impl/DatabricksConnectionContext.java that does all the properties parsing.

Reply:
I'm not sure what you want here... Cause this made sense for old proprietary driver, and is not actual for the OSS.
And you're not alone, my IP is also banned.

Reply:
Got it. Well, as we can't paste link... At least I left a comment.